Makefile

```diff
@@ -0,0 +1,10 @@
+
+
+coverage:
+	mkdir -p build/coverage
+	gcc -I. -g -fprofile-arcs -ftest-coverage tests/benzina/bcachefs/bcachefs_tests.c bcachefs/bcachefs.c -lgcov -o build/bcachefs_tests
+	./build/bcachefs_tests testdata/mini_bcachefs.img
+	mv *.gcda *.gcno build/coverage/ 
+	gcovr -r . -e tests build/coverage/
+	gcovr -r . -e tests build/coverage/ --html-details -o build/coverage/report.html
```

Comment:
Can we / should we trigger the codecov from the pytest tests?

Reply:
Ah so it's not likely that it would be possible to generate the cov files for the C code during a `pytest-cov` execution or does it need to be separate? If we could have it linked to python tests that seams to me like a cleaner way to cov the C code

Reply:
I am looking into it